src/Sylius/Bundle/ResourceBundle/Form/Type/AbstractResourceType.php

```diff
@@ -30,13 +30,20 @@
     protected $validationGroups = array();
 
     /**
+     * @var string
+     */
+    protected $name;
+
+    /**
      * @param string   $dataClass        FQCN
      * @param string[] $validationGroups
+     * @param string $name
      */
-    public function __construct($dataClass, array $validationGroups = array())
+    public function __construct($dataClass, array $validationGroups = array(), $name = 'form')
```

Comment:
@pjedrzejewski  Are you confirm? :) We have a lot of files to change, i'll get start.